### ISSUE TITLE
Update the ALThemeTemplate page

### ISSUE THREAD

**Issue by nonprofittechy** [SuffolkLITLab/docassemble-AssemblyLine-documentation#170]
~~Should cover the branding basics, how to pull and create a new repo from ALGenericJurisdiction, etc.~~
~~#32, #33, #34~~

[See checklist, below.](#issuecomment-2387034317)

**Comment by samglover** [comment]
It seems like the [customization page](https://suffolklitlab.org/docassemble-AssemblyLine-documentation/docs/get_started/customization) addresses this? Although I think that page should probably be moved under Documentation and retitled to ALThemeTemplate, at least in the sidebar.

**Comment by nonprofittechy** [comment]
Yes, that page is a big out of date as it doesn't reference the dashboard/more reliable way to use the bootstrap.build website

**Comment by samglover** [comment]
The ALThemeTemplate (f/k/a Customization) page is out of date.

## To do

- [ ] How to use ALDashboard's Bootstrap theme tool
- [ ] How to edit the organization intro screen
- [ ] Review existing content for accuracy

**Comment by plocket** [comment]
> How to edit the organization intro screen

Is that the intro screen to each interview? A home page for the organization as a whole?
